src/Validator/Constraints/EventLongerThan10MinutesSlotValidator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Validator\Constraints;
+
+use App\Dto\SlotRequest;
+use Symfony\Component\Form\Form;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+final class EventLongerThan10MinutesSlotValidator extends ConstraintValidator
+{
+    public function validate($value, Constraint $constraint): void
```

Comment:
missing type value

Reply:
Can't add value type here because it's an inherited function of ConstraintValidatorInterface